New package: FlipGraphs v0.1.1

JuliaRegistrator opened the pull request:
- Registering package: FlipGraphs
- Repository: https://github.com/schto223/FlipGraphs.jl
- Created by: @schto223
- Version: v0.1.1
- Commit: 4b5935a1601e8a9307825e5fc08c8aded343a73e
- Reviewed by: @schto223
- Reference: https://github.com/schto223/FlipGraphs.jl/issues/5#issuecomment-2126993429
- Description: My Master Thesis on Flip Graphs
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- only removed [deps] that are not needed and put them as [extras]
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598f83eb1f0956d6e4968591e1f385c32168bdc22f071656807469d54af45abb956d087877dd743f7545d37995341223c9ef6320817d41c270505dfb277879431b8d98f8ebc2266a2d44b87c327584529a25ef17ce2ff29ebe8798dac081cfb472193704586cdf867b03d5ad0be490995c273280a9bf5668da8c29853adec2dbac0 -->

gdalle commented:
Hi and congrats on the registration! Is "flip graph" a commonly accepted term in the literature? It's the first I'm hearing about it

schto223 commented:
> Hi and congrats on the registration! Is "flip graph" a commonly accepted term in the literature? It's the first I'm hearing about it

Yes, as far as I know, flip graph is the common term. I have not yet found another term for this mathematical object.
Here is a short wikipedia article on this topic: https://en.wikipedia.org/wiki/Flip_graph

gdalle commented:
Okay, thanks for the pointer! I was checking because sometimes people register packages that are not useful outside of their own research work. But if you think your package can be used by other people, and if you add some documentation (or at least a temporary explanation in the readme), I see no reason to object here :)

schto223 commented:
> https://en.wikipedia.org/wiki/Flip_graph

Okay, thank you. :) I have finally gotten the documentation up and running. And I've added a link to the former mentioned Wikipedia article into the README file.

gdalle commented:
[noblock]
Awesome, thanks! I added [noblock] to all my comments in order for the registration to proceed, you should to the same :)

JuliaRegistrator committed:
New package: FlipGraphs v0.1.1

UUID: bf6bd3dc-c349-4a39-9695-9ea99bf10cfe
Repo: https://github.com/schto223/FlipGraphs.jl.git
Tree: c2d5c614c94b148e295a8940d51969f3635c30c5

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
